WordPress/gutenberg
# Add react-native-context-menu-view library

## What?
Adds support for using Context Menus within the mobile editor. 

## Why?
To support usage of the [iOS UIMenu](https://developer.apple.com/documentation/uikit/uimenu) component, and eventually replace the usage of ActionSheet with Context Menu.

## How?
Implements the [react-native-context-menu-view](https://github.com/mpiannucci/react-native-context-menu-view) library. 

## Testing Instructions
1. Apply the demonstration context menu patch [gist: context-menu.patch](https://gist.github.com/derekblank/f85eb4f7b996dcd7bcc26d20a03c9834) (`git apply context-menu.patch`)
2. Add an Image block and tap "Add Image" 
3. Observe Context Menu (note: these Media Picker options are non-functional for visual demonstration purposes, and is not the intended implementation architecture for this component. A separate implementation involving Media Upload and Media Placeholder will be required.)

## Screenshots or screencast 
A demonstration using a mocked Media Picker as an example:

https://github.com/WordPress/gutenberg/assets/643285/d91d966a-6f59-4dee-8dc1-170b4aca43ab
